tasks/reloadOnChanges.js

```diff
@@ -0,0 +1,64 @@
+const { Task, Stopwatch } = require('klasa');
+const { watch } = require('chokidar');
+const { extname, basename, sep } = require('path');
+
+const nodeModules = `${sep}node_modules${sep}`;
+
+module.exports = class extends Task {
+
+	async run(name, _path, piece) {
+		const timer = new Stopwatch();
+
+		for (const module in require.cache) {
```

Comment:
I really think this should be a `for (const module of Object.keys(require.cache))` instead of a for-in loop